azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/DeployMojo.java

```diff
@@ -35,14 +35,14 @@
  */
 @Mojo(name = "deploy", defaultPhase = LifecyclePhase.DEPLOY)
 public class DeployMojo extends AbstractFunctionMojo {
-    public static final String FUNCTION_DEPLOY_START = "Starting deployment to Azure Function App ";
+    public static final String FUNCTION_DEPLOY_START = "Starting deployment to function app...";
```

Comment:
If you can decide whether plural form needs to be used, that's good. If not, use the following.
Deploying function app(s)...